integration-tests/initiateDataRequest.spec.ts

```diff
@@ -81,6 +121,78 @@ describe('Submit a PII request with approved ticket data', () => {
           50
         )
       expect(processDataRequestEvents).not.toEqual([])
+
+      assertEventPresent(
+        processDataRequestEvents,
+        STANDARD_TIER_OBJECTS_TO_COPY_MESSAGE
+      )
+      assertEventPresent(processDataRequestEvents, S3_COPY_JOB_STARTED_MESSAGE)
+
+      const copyCompletedEvents =
+        await getCloudWatchLogEventsGroupByMessagePattern(
+          PROCESS_DATA_REQUEST_LAMBDA_LOG_GROUP,
+          [COPY_COMPLETE_MESSAGE, 'zendeskId', ticketId],
+          100
+        )
+      expect(copyCompletedEvents).not.toEqual([])
+
+      assertEventPresent(copyCompletedEvents, NOTHING_TO_COPY_MESSAGE)
+      assertEventPresent(copyCompletedEvents, COPY_COMPLETE_MESSAGE)
+    })
+  })
+
+  describe('valid requests - data present in analysis bucket', () => {
+    let ticketId: string
+
+    beforeEach(async () => {
+      await copyAuditDataFromTestDataBucket(
+        AUDIT_BUCKET_NAME,
+        INTEGRATION_TEST_DATE_PREFIX,
+        '01',
+        TEST_FILE_NAME
+      )
+      await copyAuditDataFromTestDataBucket(
+        ANALYSIS_BUCKET_NAME,
+        INTEGRATION_TEST_DATE_PREFIX,
+        '01',
+        TEST_FILE_NAME
+      )
+      ticketId = await createZendeskTicket(validRequestData)
+      await approveZendeskTicket(ticketId)
+    })
+
+    afterEach(async () => {
+      await deleteZendeskTicket(ticketId)
+      await deleteAuditData(
```

Comment:
I would do the delete at the start of the beforeEach hook incase someone kills a test mid execution.

Reply:
Done